front/app/components/admin/Section/index.tsx

```diff
@@ -27,6 +27,18 @@ export const SectionField = styled.div`
   `}
 `;
 
```

Comment:
Added another `SectionField` that has the same width as the `PagesShowPage`, which is the component that renders pages like `/pages/faq`. This way, the form is the same width as the page itself, which pages it easier to see what the page is going to look like while you're editing it.

Reply:
Nice. Makes me think of the QuillEditedContent component, maybe we could merge or group those ?

Reply:
Can you elaborate a bit? `QuillEditedContent` is a very general component for outputting text, right?